Makefile

```diff
@@ -110,6 +111,63 @@ clean :
 	# show running pods
 	@kubectl get po -A
 
+lr-local-emul-api-only:
+	# Create LodeRunner.API image from codebase
+	@#docker build ./src -t k3d-registry.localhost:5000/loderunner-api:local -f ./src/LodeRunner.API/Dockerfile
+	# Load new LodeRunner.API image to k3d registry
+	@#docker push k3d-registry.localhost:5000/loderunner-api:local
+
+	# Delete previous deployed LodeRunner apps
+	-kubectl delete -f deploy/loderunner/local-comos/3-loderunner-api.yaml --ignore-not-found=true
```

Comment:
Update to:

-kubectl delete -f deploy/loderunner/local-cosmos/3-loderunner-api.yaml --ignore-not-found=true